<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uiDialogButtonTrans</class>
 <widget class="QDialog" name="uiDialogButtonTrans">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>404</y>
     <width>1031</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="m_pButtonHelp">
      <property name="text">
       <string>&amp;Help</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="m_pButtonNoTransition">
      <property name="text">
       <string>&amp;No Transition</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="buttonOk">
      <property name="text">
       <string>&amp;OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonCancel">
      <property name="text">
       <string>&amp;Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTabWidget" name="m_pTabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>791</width>
     <height>381</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Manual</string>
    </attribute>
    <widget class="QCheckBox" name="m_pCheckUserDefined">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>10</y>
       <width>741</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>User defined transition</string>
     </property>
    </widget>
    <widget class="QFrame" name="m_pFrameUserDefined">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>781</width>
       <height>112</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;p align=&quot;center&quot;&gt;&lt;b&gt;Here you can provide your own transition file.
This video will be played before going to the actual target of the button.
&lt;br&gt;Note: The video has to be in a proper DVD format.
&lt;br&gt;&amp;nbsp;&lt;/b&gt;&lt;/p&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="m_pEditUserFile"/>
        </item>
        <item>
         <widget class="QPushButton" name="m_pButtonBrowse">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>809</x>
     <y>39</y>
     <width>221</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="m_pLabelName">
      <property name="text">
       <string>&lt;p align=&quot;center&quot;&gt;&lt;b&gt;Transition Name&lt;/b&gt;&lt;/p&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="m_pLabelPreview">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="m_pButtonAnim">
         <property name="text">
          <string>A
n
i
m</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Filter Delay</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="m_pSpinFilterDelay">
        <property name="value">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Sec</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="m_pButtonSelectAll">
        <property name="text">
         <string>Select All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_pButtonClearAll">
        <property name="text">
         <string>Clear All</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
